index.html

```diff
@@ -4633,26 +4633,112 @@ <h2>Key and Signature Expiration</h2>
+version of a DID Document.
+      </p>
+  
+      <p>
+        If a verification method is no longer exclusively accessible to the
+        controller or parties trusted to act on behalf of the controller, it
+        should be revoked immediately to reduce the risk of masquerading, theft
+        and fraud.
+      </p>
+  
+      <p class="advisement">
+It is considered a best practice to support key revocation.
+      </p>
+  
+      <p class="advisement">
+A controller should immediately revoke any verification method that is believed to
+be compromised.
+      </p>
+  
+      <p class="advisement">
+Verifiers may choose to accept or reject proofs or signatures originating 
+from revoked verification methods, at their own discretion.
+      </p>
+  
+      <p class="note">
+As described in Section <a href="#verification-method-types"></a>,
+absence of a verification method is the only form of revocation that
+applies to all DID Methods.
+      </p>
+  
+      <p>
+Section <a href="#method-operations"></a> specifies the <a>DID</a>
+operations to be supported by a <a>DID method</a> specification,
+including <a href="#update">update</a> and <a href="#deactivate">
+deactivate</a> which MAY be used to remove verification method from
+a DID Document.
+      </p>
+
+      <p>
+Not all DID Methods support verification method revocation.
+      </p>
+  
+      <p>
+Even if a verification method is present in a DID Document, additional
+information, such as a public key revocation certificate, or an external
+allow or deny list, might be used to determine whether a verification
+method has been revoked.
+      </p>
+  
+      <p class="note">
+Compromise of a verification method allows the attacker access to the
+verification relationship of the controller, for example authentication.
+The attacker might not be indistinguishable from the legitimate
+controller from the time the compromised verification method was
+registered, to the time it was revoked.
+      </p>
+
+      <p class="note">
+The day-to-day operation of any software relying on a compromised verification
+method, such as a user’s operating system, antivirus, or endpoint protection 
+software, might be impacted when the verification method is publicly revoked.
+      </p>
+
+<p class="note">
+Although verifiers may choose not to accept proofs or signatures from revoked verification method, 
+knowing whether a verification was made with a revoked verification method is trickier than it may seem.
+Some DID methods provide the ability to look back at the state of a DID at a point in time, 
+or at a particular version of the DID document. When such a feature is 
+combined with a reliable way to determine the time or DID version that existed 
+when a cryptographically verifiable statement was made, then revocation 
+does not undo that statement. This can be the basis for using DIDs to make 
+binding commitments (e.g., to sign a mortgage). 
+<br/>
```

Comment:
```suggestion
<br/><br/>
```
Necessary for a blank line between paragraphs, unless using <p> for each (which might be better -- i.e., changing this from <p class="note"> with multiple <br/><br/> within, to <div class="note"> with multiple <p> within)

Reply:
Also, side note that `<br>` is a [void element](https://www.w3.org/TR/html52/syntax.html#void-elements) so you can just write `<br><br>` here, no need to the closing slash.

Reply:
I have updated @TallTed's change request to match @dlongley's request.